File: src/main/java/no/ntnuf/towlog/towlog2/common/TowEntry.java
package no.ntnuf.towlog.towlog2.common;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by Christian on 21.10.2015.
 */
public class TowEntry implements Serializable {

    private static final long serialVersionUID = 4L;

    public int height = 0;

    public Contact pilot;
    public Contact copilot;

    public Date towStarted;

    public String registration = "";
    public String notes = "";

    public String gpx_filename = null;
    public String gpx_body = null;

}

File: app/src/main/java/no/ntnuf/tow/towlog2/model/TowEntry.kt
package no.ntnuf.tow.towlog2.model

import java.io.Serializable
import java.util.Date

data class TowEntry(
    val height: Int = 0,
    val pilot: Contact,
    val copilot: Contact? = null,
    val towStarted: Date,
    val registration: String = "",
    val notes: String = "",
    val gpx_filename: String? = null,
    val gpx_body: String? = null
) : Serializable {

    companion object {
        private const val serialVersionUID = 1L
    }
}
